app/workers/reduce_worker.rb

```diff
@@ -22,8 +22,11 @@ def self.unique_args(args)
 
   def perform(reducible_id, reducible_class, subject_id, user_id, extract_ids = [])
     reducible = reducible_class.constantize.find(reducible_id)
-    return if reducible.class.name.demodulize=='Workflow' && reducible.paused?
+    return if reducible.respond_to?('paused?') && reducible.paused?
```

Comment:
Why not just implement `dcf paused?; false; end` on Project?

Reply:
good idea, that's a lot simpler